app/controllers/concerns/paging_concern.rb

```diff
@@ -28,20 +29,21 @@ def pagination(collection, path_resolver: :paging_path_for, url_format: nil)
 
     format_type = url_format || self.paging_url_format
     total_items = collection.count
-    total_pages = total_items.zero? ? 1 : (total_items / PAGE_SIZE.to_f).ceil
-    page_number = (params[:page] || 1).to_i.clamp(1, total_pages)
-    page_offset = (page_number - 1) * PAGE_SIZE
+    page_size = (params[:chunk_size] || DEFAULT_PAGE_SIZE).to_i.clamp(1, MAX_PAGE_SIZE)
+    total_pages = total_items.zero? ? 1 : (total_items / page_size.to_f).ceil
+    page_number = (params[:chunk] || 1).to_i.clamp(1, total_pages)
+    page_offset = (page_number - 1) * page_size
```

Comment:
I know it’ll be more churn, but should we rename these local variables (and the module constants at the top of the file) to be about `chunk` instead of `page`?

Reply:
Sure. I was being uncharacteristically gun shy. I'm a fan of internal consistency in local var names.